cps/src/test/java/org/jenkinsci/plugins/workflow/cps/SnippetizerTest.java

```diff
@@ -216,4 +216,27 @@ private void assertGenerateSnippet(@Nonnull String json, @Nonnull String respons
         assertThat("content is written to the end", html, containsString("</body></html>"));
     }
 
+    @Issue("JENKINS-26126")
+    @Test public void doGdsl() throws Exception {
+        JenkinsRule.WebClient wc = r.createWebClient();
+        String gdsl = wc.goTo(Snippetizer.GDSL_URL, "text/plain").getWebResponse().getContentAsString();
+        assertThat("Description is included as doc", gdsl, containsString("Build a job"));
+        assertThat("Timeout step appears", gdsl, containsString("name: 'timeout'"));
+
+        // Verify valid groovy syntax.
+        GroovyShell shell = new GroovyShell(r.jenkins.getPluginManager().uberClassLoader);
+        shell.parse(gdsl);
+    }
+
+    @Issue("JENKINS-26126")
+    @Test public void doDsld() throws Exception {
+        JenkinsRule.WebClient wc = r.createWebClient();
+        String dsld = wc.goTo(Snippetizer.DSLD_URL, "text/plain").getWebResponse().getContentAsString();
+        assertThat("Description is included as doc", dsld, containsString("Build a job"));
+        assertThat("Timeout step appears", dsld, containsString("name: 'timeout'"));
+
+        // Verify valid groovy sntax.
+        GroovyShell shell = new GroovyShell(r.jenkins.getPluginManager().uberClassLoader);
+        shell.parse(dsld);
```

Comment:
DSLD and GDSL are both Groovy - the output's different, but still Groovy in both cases.